3073a17202b0c9687d6302ffbfb10c2363169448..fe9799314b30e801094a6cfef8a15bf403857285
50fdd5e2b5c6347ccffb19efe31f271feca92de0..ce5477ef02688e4f535c6916c26193743ab38f40

commit 2f03795cb0b0550f26c0040da64cc918c7a03539
Author:     Robert Walker <bob.mt.wya@gmail.com>
AuthorDate: Sun Jan 29 11:56:51 2023 +0000
Commit:     Robert Walker <bob.mt.wya@gmail.com>
CommitDate: Sun Jan 29 11:56:51 2023 +0000

winecfg: display full package string
---

 programs/winecfg/about.c    | 9 ++++++---
 programs/winecfg/resource.h | 1 -
 programs/winecfg/winecfg.rc | 8 ++++----
 3 files changed, 10 insertions(+), 8 deletions(-)


diff --git a/programs/winecfg/about.c b/programs/winecfg/about.c
index fb604dda184..649263eb8cc 100644
--- a/programs/winecfg/about.c
+++ b/programs/winecfg/about.c
@@ -46,4 +46,6 @@ AboutDlgProc (HWND hDlg, UINT uMsg, WPARAM wParam, LPARAM lParam)
     RECT rcClient, rcRect;
     char *owner, *org;
+    static CHAR package_string[256];
+
 
     switch (uMsg)
@@ -109,10 +111,12 @@ AboutDlgProc (HWND hDlg, UINT uMsg, WPARAM wParam, LPARAM lParam)
 
         /* prepare the title text */
-        titleFont = CreateFontW( -MulDiv(24, GetDeviceCaps(hDC, LOGPIXELSY), 72),
+        titleFont = CreateFontW( -MulDiv(16, GetDeviceCaps(hDC, LOGPIXELSY), 72),
                                  0, 0, 0, 0, FALSE, 0, 0, 0, 0, 0, 0, 0, tahomaW );
         SendDlgItemMessageW(hDlg, IDC_ABT_TITLE_TEXT, WM_SETFONT, (WPARAM)titleFont, TRUE);
 
         wine_get_version = (void *)GetProcAddress( GetModuleHandleA("ntdll.dll"), "wine_get_version" );
-        if (wine_get_version) SetDlgItemTextA(hDlg, IDC_ABT_PANEL_TEXT, wine_get_version());
+    lstrcpyA(package_string, "");
+    if (wine_get_version) lstrcatA(package_string, wine_get_version());
+    SetDlgItemTextA(hDlg, IDC_ABT_TITLE_TEXT, package_string);
 
         ReleaseDC(hDlg, hDC);
@@ -163,5 +167,4 @@ AboutDlgProc (HWND hDlg, UINT uMsg, WPARAM wParam, LPARAM lParam)
             SetBkColor((HDC)wParam, GetSysColor(COLOR_WINDOW));
             return (INT_PTR)GetSysColorBrush(COLOR_WINDOW);
-        case IDC_ABT_PANEL_TEXT:
         case IDC_ABT_LICENSE_TEXT:
         case IDC_ABT_WEB_LINK:
diff --git a/programs/winecfg/resource.h b/programs/winecfg/resource.h
index 2760e92c000..46c6da26726 100644
--- a/programs/winecfg/resource.h
+++ b/programs/winecfg/resource.h
@@ -216,5 +216,4 @@
 #define IDC_ABT_ORG                  8433
 #define IDC_ABT_PANEL                8434
-#define IDC_ABT_PANEL_TEXT           8435
 #define IDC_ABT_TITLE_TEXT           8436
 #define IDC_ABT_WEB_LINK             8437
diff --git a/programs/winecfg/winecfg.rc b/programs/winecfg/winecfg.rc
index f2f795f5278..aac6b9310eb 100644
--- a/programs/winecfg/winecfg.rc
+++ b/programs/winecfg/winecfg.rc
@@ -21,4 +21,5 @@
  */
 
+#include "config.h"
 #include "resource.h"
 
@@ -134,9 +135,8 @@ FONT 8, "MS Shell Dlg"
 BEGIN
     CONTROL         "", IDC_ABT_PANEL, "STATIC", SS_OWNERDRAW, 0, 0, 260, 140
-    LTEXT           "Wine",IDC_ABT_TITLE_TEXT,105,30,55,30
-    LTEXT           "",IDC_ABT_PANEL_TEXT,160,43,140,8
-    CONTROL         "#msgctxt#do not translate#<a href=\"https://www.winehq.org\">https://www.winehq.org</a>",IDC_ABT_WEB_LINK,"SysLink", LWS_TRANSPARENT, 105,53,106,8
+    LTEXT           "",IDC_ABT_TITLE_TEXT,100,30,200,20
+    CONTROL         "#msgctxt#do not translate#<a href=\"https://www.winehq.org\">https://www.winehq.org</a>",IDC_ABT_WEB_LINK,"SysLink", LWS_TRANSPARENT, 100,53,106,8
     LTEXT           "This program is free software; you can redistribute it and/or modify it under the terms of the GNU Lesser General Public License as published by the Free Software Foundation; either version 2.1 of the License, or (at your option) any later version.",
-                    IDC_ABT_LICENSE_TEXT,105,64,145,66
+                    IDC_ABT_LICENSE_TEXT,100,64,145,66
     GROUPBOX        "Windows registration information", IDC_STATIC, 15, 155, 230, 55
     LTEXT           "&Owner:", IDC_STATIC, 22, 171, 40, 20
--
